### PR TITLE
Feature/eui 2891 post task

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/work-allocation/services/work-allocation-task.service.ts [hunk 1]
```diff
@@ -14,7 +15,14 @@ export class WorkAllocationTaskService {
    * @param taskId specifies which task should be completed.
    */
   public completeTask(taskId: string): Observable<any> {
-    // Make a POST with an empty payload.
     return this.http.post<any>(`${BASE_URL}${taskId}/complete`, {});
   }
+
+  public postTask(task: TaskSearchParameters): Observable<any> {
+    return this.http.post<any>(`${BASE_URL}`, {});
```
Comment (paul-graham) [inline src/work-allocation/services/work-allocation-task.service.ts:22]: Doesn't this need to pass the search parameters in the body?

**Review by paul-graham — CHANGES_REQUESTED** [review]

File: src/work-allocation/services/work-allocation-task.service.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/work-allocation/services/work-allocation-task.service.ts:22]: yes

**Review by paul-graham — APPROVED** [review]
Looks good to me.

**Review by sreekanthpuligadda — COMMENTED** [review]
looks good

**Pull request merged**